src/Bundle/test/src/Migration/SyliusAdminOrder.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+/**
+ * This code is generated by the config converter under https://github.com/mamazu/grid-config-converter
+ * Feel free to modify the code as you see fit.
+ */
+
+namespace SomeNameSpace;
+
+use Sylius\Bundle\GridBundle\Builder\Action\ShowAction;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\ItemActionGroup;
+use Sylius\Bundle\GridBundle\Builder\Field\DateTimeField;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\Field\TwigField;
+use Sylius\Bundle\GridBundle\Builder\Filter\Filter;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+class SyliusAdminOrder extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public static function getName(): string
+    {
+        return 'sylius_admin_order';
+    }
+
+    public function getResourceClass(): string
+    {
+        return 'Sylius\\Model\\Order';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            ->setDriver('doctrine/orm')
+            ->setRepositoryMethod('myCustomMethod', [
+                'id' => 'resource.id',
+            ])
+            ->setDriverOption('class', '%sylius.model.order.class%')
+            ->setDriverOption('pagination', [
+                'fetch_join_collection' => false,
+                'use_output_walkers' => false,
+            ])
+            ->addOrderBy('number', 'desc')
+            ->setLimits([
+                30,
+                12,
+                48,
+            ])
+            ->addField(
+                DateTimeField::create('date')
+                ->setEnabled(false)
+                ->setLabel('sylius.ui.date')
+                ->setPosition(100)
+                ->setPath('checkoutCompletedAt')
+                ->setSortable(true)
+                ->setOptions([
+                    'format' => 'd-m-Y H:i:s',
+                ]),
+            )
+            ->addField(
+                TwigField::create('number', '@SyliusAdmin/Order/Grid/Field/number.html.twig')
+                ->setLabel('sylius.ui.number')
+                ->setPath('.')
+                ->setSortable(true)
+                ->setOptions([
```

Comment:
use addOption instead, it's safer to prevent overriding defaultOptions (as you already have on TwigField). But maybe we have to fix this before...